SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -201,8 +202,17 @@ public SingularityRequestParent bounce(@ApiParam("The request ID to bounce") @Pa
 
     requestManager.bounce(requestWithState.getRequest(), System.currentTimeMillis(), JavaUtils.getUserEmail(user), message);
 
+    SingularityBounceRequest defaultBounceRequest = bounceRequest.or(SingularityBounceRequest.defaultRequest());
+    if (!defaultBounceRequest.getDurationMillis().isPresent()) {
+      final Long durationMillis = TimeUnit.MINUTES.toMillis(configuration.getDefaultBounceExpirationMinutes());
```

Comment:
In this case I was following the convention established in the constructor of the `SingularityBounceRequest` class (`@JsonProperty("durationMillis") Optional<Long> durationMillis`).  If I change it here, should I change it in the constructor as well?